### PR TITLE
Logic for cat 1 and 2 - Entry and exit - exemptions questions pages 

### PR DESCRIPTION
**Pull request by alexdesi**
### Jira link
https://transformuk.atlassian.net/browse/GL-664
https://transformuk.atlassian.net/browse/GL-723

### What?

-[x] It adds the Entry logic for the Cat1 and Cat2 exemption questions
-[x] It adds the Exit logic for the Cat1 and Cat2 exemption questions
-[x] It adds placeholders for the new Result page 1,2,3

### Why?
This is an Iteration to implement the final design in the
https://miro.com/app/board/uXjVKGApxh8=/?moveToWidget=3458764592795747525&cot=14

### CODE REVIEW

File: app/services/green_lanes/determine_category.rb [hunk 1]
```diff
@@ -51,7 +53,7 @@ def category_assessments(category = nil)
 
       return ca_assessments unless category
 
-      category_assessments.select { |ca| ca.theme.category == category }
```
Comment (shjohnson) [inline app/services/green_lanes/determine_category.rb:54]: 👍

File: app/services/green_lanes/determine_next_page.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  class DetermineNextPage
+    def initialize(goods_nomenclature)
+      @categories = GreenLanes::DetermineCategory.new(goods_nomenclature).categories
+    end
+
+    def next(cat_1_exemptions_apply: nil, cat_2_exemptions_apply: nil)
+      case @categories
+      when [:cat_1], [:cat_2], [:cat_3]
+        # Simplest case: only one category is present
+        "result_#{@categories.first}".to_sym
```
Comment (shjohnson) [inline app/services/green_lanes/determine_next_page.rb:11]: will this always be in the same order?

**Comment by shjohnson** [comment]
I think mostly look fine was only thinking if we wanted to change the `:cat_x` symbols to be `:res_x` in the determine category?

File: app/services/green_lanes/determine_next_page.rb (hunk 2 above)
Comment (alexdesi) [reply app/services/green_lanes/determine_next_page.rb:11]: @categories is an array containing the :cat_1, :cat_2, :cat_3

**Review by shjohnson — APPROVED** [review]

**Pull request merged**